config/secrets.yml

```diff
@@ -36,9 +36,9 @@ production:
   app_name: CAS Shipit
   secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
   host: <%= ENV['SHIPIT_HOST'] %>
-  redis_url: redis://:<%= ENV['REDIS_PASSWORD'] %>@<%= ENV['REDIS_HOST'] %>:<%= ENV['REDIS_PORT'] || 6379 %>
+  redis_url: <%= ENV['REDIS_URL'] %>
```

Comment:
We'll need to change on the cluster how the URL is fetched (or rebuild it like before and cancel this line change)